docs/reports/report_27_11.md

```diff
@@ -0,0 +1,33 @@
+# Raport ze spotkania
+
+## Opis dokumentu
+ Dokument zawiera notatki ze spotkania z klientem dnia 27-11-2021.
+
+## Osoby obecne na spotkaniu
+
+- Mateusz Tylka - **twórca Wmi Adventure**
+- Marcin Kostrzewski - **twórca Wmi Adventure**
+- Michał Czekański - **twórca Wmi Adventure**
+- Ola (studentka drugiego roku informatyki na Wmi) - **klient**
+
+## Proces spotkania
+1. Przedstawienie funkcji produktu.
+2. Przeprowadzenie wywiadu z klientem na temat jakości produktu.
+
+## Opinia klienta po prezentacji
+
+Opinia klienta była bardzo pozytywna. Klient za pierwszy razem zrozumiał konwencję projektu oraz mechanikę walki.

```

Comment:
Ja sobie zapisałem sobie takie rzeczy:
- według klient kolorystyka i design są przyjemne
- w walce karty za szybko leciały, ciężko było przeczytać co robi dana karta
- podobało się to, że nie tylko efekty kart mają znaczenie, ale też kolejność kart
- pomysł na bossy nawiązujące do prowadzących na uczelni też się podobał